etc/deploy/aws.sh

```diff
@@ -1,18 +1,27 @@
 #!/bin/bash
 
+# Argument Defaults
 METRICS_FLAG="true"  # By default, aws.sh enables metric reporting
+USE_CLOUDFRONT="false"
+
+# Other defaults
+CLOUDFRONT_DOMAIN=
 
 set -euxo pipefail
 
 parse_flags() {
+  # Check prereqs
+  which aws
+  which jq
+  which uuid
   # Common config
   export AWS_REGION=us-east-1
   export AWS_AVAILABILITY_ZONE=us-east-1a
   export STATE_BUCKET=s3://k8scom-state-store-pachyderm-${RANDOM}
   local USE_EXISTING_STATE_BUCKET='false'
 
   # Parse flags
-  eval "set -- $( getopt -l "state:,region:,zone:,no-metrics" "--" "${0}" "${@}" )"
+  eval "set -- $( getopt -l "state:,region:,zone:,no-metrics:,use-cloudfront" "--" "${0}" "${@}" )"
```

Comment:
Now that I've read up on getopts I realize `no-metrics` shouldn't have a trailing colon. Will fix.